doc/user/interop.md

```diff
@@ -67,7 +67,8 @@ produces the address as a `long`. For all other objects calls `unbox`.
 
 ### `IS_POINTER`
 
-Returns true only for `FFI::Pointer`.
+Calls `pointer?` if the object responds to it, otherwise throws
```

Comment:
I think according to Truffle interop semantics, `IS_POINTER` is never supposed to throw. I think it would be better to return `false` if the object doesn't respond to `pointer?`.

Reply:
Looking at the code below, this seems to be the case, so this is just wrong documentation.

Reply:
Good catch. I had it throw and then changed it to return `false` when I realized that was causing a ton of deopts.